sdk/cosmos/azure-cosmos/azure/cosmos/_synchronized_request.py

```diff
@@ -90,10 +90,7 @@ def _Request(global_endpoint_manager, request_params, connection_policy, pipelin
     """
     # pylint: disable=protected-access
 
-    is_media = request.url.find("media") > -1
-    is_media_stream = is_media and connection_policy.MediaReadMode == documents.MediaReadMode.Streamed
-
-    connection_timeout = connection_policy.MediaRequestTimeout if is_media else connection_policy.RequestTimeout
```

Comment:
Could we remove `MediaRequestTimeout` from the connection policy object and documentation?
Some other places to remove:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py#L80
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py#L82
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py#L133
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py#L136
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/documents.py#L341
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cosmos/azure-cosmos/azure/cosmos/documents.py#L347